test/test_turn_integrator.py

```diff
@@ -10,7 +10,7 @@ def load_test_config():
 
 def test_eval_credentials():
```

Comment:
@DylanIvanBrown I find VCR is a great way to handle this. So you make an API call once and record that, then play back the cassettes until you delete them, after which the functionality you are testing will make API call again. I find this a better way to test than to mock things out, because your tests will have an actual API interaction every now and then, if you want.